services/nuget/nuget-v3-service-family.js

```diff
@@ -76,7 +76,7 @@ async function fetch(
   serviceInstance,
   { baseUrl, packageName, includePrereleases = false }
 ) {
-  const json = await serviceInstance._requestJson({
+  return await serviceInstance._requestJson({
```

Comment:
No need for the await here since processing of the http response is no longer being done within this async function

```suggestion
  return serviceInstance._requestJson({
```

Reply:
Ha, I think I don't get used to the `async` quirks of JS. Too much used to C#/Rust.

Fixed!